types/office-js-preview/index.d.ts

```diff
@@ -25022,7 +25022,7 @@ declare namespace Excel {
     interface BasicViewLayouts {
         /**
          * Represents the layout of this value and its properties in card view.
-         * If the `BasicCardLayout` object doesn't have a layout property, it's assumed to be "Entity".
+         * If the `BasicCardLayout` object does not have a layout property, it is assumed to be "Entity".
```

Comment:
Note for reviewers: This PR has mostly trivial whitespace and contraction changes. 

If you see a contraction *removed* in this PR, it's because the contraction doesn't exist in the source code and I've decided to leave it that way.